frontend/packages/console-shared/src/components/popover/Popover.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { Popper } from '../popper';
+import * as FocusTrap from 'focus-trap-react';
```

Comment:
Use focus trap from PF:
```suggestion
import * as FocusTrap from '@patternfly/react-core';
```

Reply:
Agree, there are only a few other places that use FocusTrap from this external lib, we should get rid of them and drop the dependency.

Reply:
fixed in other places as well

Reply:
Not sure if we want to remove the dependency as well from package.json

Reply:
@christianvogt Did you want to consult with the Admin team?

@sahil143 Did you test the other areas? Because you've just created an extra step before I can lgtm this lol... Will have to wait until I can test the other areas.